apps/next-evans/lib/seo.ts

```diff
@@ -0,0 +1,32 @@
+import { inflect } from './utils';
+
+export function friendPageDescription(
```

Comment:
hmmm... are you handling spanish books here at all?

or, i guess, maybe at a deeper level, did you decide against just pulling over my `lib/seo.ts` file? all this lang-aware string concat stuff is pretty grungy, and it's all already done (with tests), so it seems like unless we had a compelling reason to rewrite, we should probably use those. sorry if i didn't make it more clear that there was code for this already.